src/validation_workflow/rpm/validation_rpm.py

```diff
@@ -25,15 +25,13 @@ def __init__(self, args: ValidationArgs) -> None:
         self.tmp_dir = TemporaryDirectory()
 
     def download_artifacts(self) -> bool:
-        architectures = ["x64", "arm64"]
         for project in self.args.projects:
-            for architecture in architectures:
-                url = f"{self.base_url}{project}/{self.args.version}/{project}-{self.args.version}-linux-{architecture}.rpm"
-                if ValidateRpm.is_url_valid(url) and ValidateRpm.download(url, self.tmp_dir):
-                    logging.info(f"Valid URL - {url} and Download Successful !")
-                else:
-                    logging.info(f"Invalid URL - {url}")
-                    raise Exception(f"Invalid url - {url}")
+            url = f"{self.base_url}{project}/{self.args.version}/{project}-{self.args.version}-linux-{self.args.arch}.rpm"
```

Comment:
If the url does not end in `/` this will fail right? Maybe add a check or add the `/` and trim the `self.base_url`

Reply:
Don't think so @gaiksaya . self.base_url is predefined [here](https://github.com/opensearch-project/opensearch-build/blob/4a6e2f2118ff7381a9709b32ae75711ea3f41494/src/validation_workflow/tar/validation_tar.py#L25)
And the string format is as per download url's for opensearch, this basically fails only when the given version does not exist.